App.js

```diff
@@ -38,14 +38,16 @@ class App extends React.Component {
   render(){
     return (
       <div>
-        <input type="textarea" name="textToAdd" id="textToAdd" size="75" defaultValue={this.props.text}/>
-        <input type="submit" value="Add" id="addbutton" onClick={this.update.bind(this)}/>
+        <input type="textarea" name="textToAdd" id="textToAdd" size="75" defaultValue={this.props.text} data-toggle="tooltip" data-placement="bottom" title="This is the new text we will add to the list!"/>
+        <input type="button" value="Add" id="addbutton" onClick={this.update.bind(this)}/>
```

Comment:
Ah! I would suggest reverting it to the way it is in `master` this way there are no merge conflicts 😃